prometheus/inc/class-plugin.php

```diff
@@ -114,11 +115,7 @@ public function process_metrics(): void {
 			return;
 		}
 
-		foreach ( $this->collectors as $collector ) {
-			if ( is_callable( [ $collector, 'process_metrics' ] ) ) {
-				$collector->process_metrics();
-			}
-		}
+		array_walk( $this->collectors, fn ( CollectorInterface $collector ) => $collector->process_metrics() );
```

Comment:
We're sure about removing the safeguard?

Reply:
Yes. `process_metrics()` is the part of the interface: https://github.com/Automattic/vip-go-mu-plugins/pull/4777/files#diff-570d0312e081d70141767fdbc5ab1cef3561f42de78e644998fdb4b1f77b0cf7R20

Reply:
Ah right, goes alongside the other change to add that back to the interface 👍